runtime/ast/statement.go

```diff
@@ -251,6 +331,36 @@ func (s *ForStatement) EndPosition() Position {
 	return s.Block.EndPosition()
 }
 
+const forStatementForKeywordSpaceDoc = prettier.Text("for ")
+const forStatementSpaceInKeywordSpaceDoc = prettier.Text(" in ")
+
+func (s *ForStatement) Doc() prettier.Doc {
+	doc := prettier.Concat{
+		forStatementForKeywordSpaceDoc,
+	}
+
+	if s.Index != nil {
+		doc = append(
```

Comment:
Is there a difference between using an array (append) vs `prettier.Concat{}`

Reply:
`prettier.Concat` is a prettier document, but is also just a slice of documents, so we can use the Go `append` to add to it